nebullvm/operations/inference_learners/neural_compressor.py

```diff
@@ -134,11 +138,12 @@ def load(cls, path: Union[Path, str], **kwargs):
         q_model = convert_fx(q_model)
 
         q_model.load_state_dict(state_dict)
+        device = Device(DeviceType(metadata.device))
```

Comment:
what about implementing a class method of Device for directly loading the string (and convert it to device type in the method)?

Something like
```python
device = Device.from_str(metadata.device)
```

Reply:
Great idea!